mmhuman3d/core/visualization/visualize_smpl.py

```diff
@@ -346,18 +344,17 @@ def _prepare_mesh(poses, betas, transl, verts, start, end, body_model):
         num_verts = body_model.NUM_VERTS
         assert verts.shape[-2] == num_verts, 'Wrong input verts shape.'
         faces = body_model.faces_tensor
-        num_frame = verts.shape[0]
-        start = (min(start, num_frame - 1) + num_frame) % num_frame
-        end = (min(end, num_frame - 1) + num_frame) % num_frame
-        verts = verts[start:end + 1]
-        num_frame = verts.shape[0]
-        vertices = verts.view(num_frame, -1, num_verts, 3)
+        num_frames = verts.shape[0]
+        start = (min(start, num_frames - 1) + num_frames) % num_frames
```

Comment:
Similar as above.

Reply:
this is still useful since in `ffmpeg`, `start_frame` and `end_frame` must be None.